src/Controls/tests/Core.UnitTests/MotionTests.cs

```diff
@@ -40,21 +42,19 @@ public void SetEnabled(bool enabled)
 
 			_enabled = enabled;
 
-			OnSystemEnabledChanged();
+			//OnSystemEnabledChanged();
```

Comment:
Was this an intentional removal, or left over from development?

Reply:
It's not needed. Deleting the line.